extensions/imports/package.json

```diff
@@ -13,7 +13,9 @@
     "import"
   ],
   "dependencies": {
-    "jscodeshift": "^0.3.0",
```

Comment:
I believe jscodeshift should still be a peer dependency. After all, the plugin depends on specific version of the API.